### PR TITLE
Docs: Iroha v1 Doc fixes

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

Small fixes in the docs: 
- links to katacoda and l10n (in README and CONTRIBUTING)
- build command fix

### Benefits

Docs ready for release

### Possible Drawbacks 

Still need to finish l10n repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,24 +27,17 @@ Iroha is _Crash Fault Tolerant_ and has its own consensus algorithm - [YAC](http
 
 ## Documentation
 
-Our documentation is hosted at ReadTheDocs service here: [http://iroha.readthedocs.io](http://iroha.readthedocs.io).
-We have documentation in several languages available and you are welcome to contribute on [POEditor website](https://poeditor.com/join/project/SFpZw7o33o)!
+Our documentation is hosted at ReadTheDocs service here: [http://iroha.readthedocs.io](http://iroha.readthedocs.io) and has different versions.
```
Comment (lebdron) [inline README.md:30]: I think that "our documentation has different version" can have several meanings. Maybe rephrase it in a more specific way?

**Review by lebdron — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (LiraLemur) [reply README.md]: Does it look better now, @lebdron ?

Comment (lebdron) [reply README.md]: Yes, thank you!
